references.bib

```diff
@@ -659,10 +662,22 @@ @online{kyotovr2018
 	urldate={2019-10-01}
 }
 
+@comment {
+    are images from wikimedia commons okay?
```

Comment:
Yes as long as they have a permissive license. If it's Creative Commons you need to mention the license and provide a link to it.
`note={Licensed under \href{https://creativecommons.org/licenses/by/2.0/}{CC BY 2.0}},'